subsys/bluetooth/controller/Kconfig

```diff
@@ -29,4 +29,24 @@ choice BLE_CONTROLLER_VARIANT
 
 endchoice
 
+
+comment "Thread safety behavior of BLE controller APIs"
+
+config BLECTLR_THREADSAFE_BLOCKING
```

Comment:
BLOCKING is misleading. Please change

Reply:
changing 
BLECTLR_THREADSAFE_BLOCKING -> BLECTLR_THREADSAFETY and 
BLECTLR_THREADSAFE_BLOCKING_TIMEOUT -> BLECTLR_THREADSAFETY_LOCK_TIMEOUT